fs/rc/cache.go

```diff
@@ -41,3 +44,37 @@ func GetFsAndRemoteNamed(in Params, fsName, remoteName string) (f fs.Fs, remote
 func GetFsAndRemote(in Params) (f fs.Fs, remote string, err error) {
 	return GetFsAndRemoteNamed(in, "fs", "remote")
 }
+
+// GetExpireAndUnlink get the `expire` and `unlink` parameters from in
+func GetExpireAndUnlink(in Params) (expire time.Duration, unlink bool, err error) {
```

Comment:
Can you move this to `fs/rc/params.go` and call it `GetDuration` and change it so it only reads a duration. This could do with a test too in `fs/rc/params_test.go`